portfolio/src/main/webapp/script.js

```diff
@@ -25,26 +25,73 @@ function changeTextBack(text) {
   display.innerHTML = text;
 }
 
+//Form Text Input Resizing Function - Name
+let elName = document.querySelector(".input-wrap-name .input");
+let extraWidthName = document.querySelector(".input-wrap-name .extra-width-name");
+elName.addEventListener("keyup", () => {
+  extraWidthName.innerHTML = elName.value;
+});
+
+//Form Text Input Resizing Function - Message
+let elMessage = document.querySelector(".input-wrap-message .input");
+let extraWidthMessage = document.querySelector('.input-wrap-message .extra-width-message');
+elMessage.addEventListener("keyup", () => {
+  extraWidthMessage.innerHTML = elMessage.value;
+});
```

Comment:
You can create a wrapper method that adds a layer of abstraction above the event listener. But looking at this now, I don't think that would bring tremendous value here. This is clearer. But if you find you're adding the same listener to multiple (more than 2 or 3) elements that may be useful to avoid code repetition.